biz.aQute.launcher/src/aQute/launcher/constants/LauncherConstants.java

```diff
@@ -94,6 +96,7 @@
 			p.setProperty(LAUNCH_NAME, name);
 
 		p.setProperty(LAUNCH_NOTIFICATION_PORT, String.valueOf(notificationPort));
+		p.setProperty(LAUNCH_ACTIVATION_EAGER, activationEager + "");
```

Comment:
Sorry... one more thing - can we not keep kicking this can down the road?
```suggestion
		p.setProperty(LAUNCH_ACTIVATION_EAGER, String.valueOf(activationEager));
```